lib/shared/utils.js

```diff
@@ -48,22 +48,15 @@ function timecodeToMicroseconds(timecode, fps) {
 }
 
 function extractStyling(text) {
-  let unformattedText = '';
-  if (text.search(/<br>/) && !text.startsWith('<br>')) {
-    unformattedText = text.replace(/<br>/, '\n');
-  }
-  const formattingTags = unformattedText.match(/<[^>]*>|{[^}]*}/g);
-  if (formattingTags && formattingTags.length) {
-    formattingTags.forEach(tag => {
-      if (tag.search(/{[^}]*}/) > -1) {
-        unformattedText = unformattedText.replace(tag, ' ').replace(/\s\s+/, ' ');
-      } else {
-        unformattedText = unformattedText.replace(tag, '');
-      }
-    });
-  }
-
-  return unformattedText.trim();
+  const regexReplace = [
+    [/(?<!^)<br>/g, '\n'],
+    [/<.*?>/g, ''],
+    [/{.*?}/g, ' '],
+    [/((?<!--)>|<|{|})/g, ''],
```

Comment:
Actually negative look behind is not necessary here.  `extractStyling` is only run on the parsed out text portion of an entry.  Timecodes with `-->` will not be affected.